src/OpenTK.GLControl/GLControl.cs

```diff
@@ -191,32 +212,37 @@ protected override void OnHandleCreated(EventArgs e)
                 implementation.WindowInfo.Dispose();
             }
 
-            if (design_mode)
-            {
-                implementation = new DummyGLControl();
-            }
-            else
+            try
             {
-                implementation = new GLControlFactory().CreateGLControl(format, this);
-            }
+                if (design_mode)
+                    implementation = new DummyGLControl();
```

Comment:
We don't use the "short if" style. Swap to explicit braces here.
```csharp
if (condition) {
    // do stuff
}
```

Reply:
Will do